README.md

```diff
@@ -1,36 +1,17 @@
 # lightsaber [![Build Status](https://travis-ci.org/captn3m0/lightsaber.svg?branch=master)](https://travis-ci.org/captn3m0/lightsaber)
 
 Lightsaber is a simple DNS Redirect service. It offers 301/302 redirects for
-your domains. The configuration is kept public on this github repository itself.
+your domains. The configuration is stored in TXT records.
 
 ## Usage
 
 To add a DNS record, point your domain via a CNAME entry to `lightsaber.captnemo.in`.
 
-You can also have fallthrough domain redirects
-(thanks to @vivekprakash for the suggestion), by making the redirect a hash
-with a `root` key set to the url prefix.
+Then create a TXT record for the `_redirect` subdomain and use the following as the value:
 
-```yaml
-# This is a fallthrough redirect
-# so t.co/ev would redirect to twitter.com/ev
-t.co:
-  root: https://twitter.com
-# All goo.gl routes will forward to google.com, and not use fallthrough
-goo.gl: https://google.com
-```
+    v=lr1;https://google.com
```

Comment:
Probably makes sense to use code=30x instead of type or status or?

As for the schema:
Recommended/default ```v=dr1;url=https://google.com;code=30x```
If a value is provided without a key it is assumed, it is assumed it is the value of the **url** key:
```v=dr1;https://google.com;code=30x```

If no code is provided, it is assumed it is a 301 redirect:
```v=dr1;url=https://google.com```

Does that seem reasonable?